### PR TITLE
use config from state

### PR DESCRIPTION
**Pull request by alichaddad**
Related to https://github.com/Jumpscale/core9/pull/83

### CODE REVIEW

File: docs/gettingstarted/startays.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ You will first want to review the configuation of your AYS server, as documented
 
 Most importantly, in order to run AYS in **production mode** you need to update or add following two sections in `/etc/jumpscale9.toml`:
 ```toml
-[ays]
+[config.ays]
```
Comment (muhamadazmy) [inline docs/gettingstarted/startays.md]: Why do we need this extra namespacing ?

Comment (rkhamis) [reply docs/gettingstarted/startays.md]: to be able to get configs seperately from states. (current installed pip modules, dirs specified as code dirs. Nodes to sync to...etc)

Comment (muhamadazmy) [reply docs/gettingstarted/startays.md]: what are the states ?
also this makes it look very weird in the code when u have to do `j.core.config['config']['ays']` no ?

Comment (alichaddad) [reply docs/gettingstarted/startays.md]: no it will be `j.core.config['ays']`.

File: JumpScale9AYS/ays/AtYourServiceFactory.py [hunk 2]
```diff
@@ -98,7 +98,7 @@ def actorTemplates(self):
     @property
     def config(self):
         if self._config is None:
-            cfg = j.application.config.get('ays')
+            cfg = j.core.state.configGet('ays')
```
Comment (rkhamis) [inline JumpScale9AYS/ays/AtYourServiceFactory.py]: no fallback? What happens when not configured?

File: docs/gettingstarted/startays.md (hunk 1 above)
Comment (rkhamis) [reply docs/gettingstarted/startays.md]: with seperate files, this needs to be updated @ashraffouda

File: docs/gettingstarted/startays.md [hunk 3]
```diff
@@ -4,10 +4,10 @@ You will first want to review the configuation of your AYS server, as documented
...
 production = true
 
-[ays.oauth]
+[config.ays.oauth]
```
Comment (rkhamis) [inline docs/gettingstarted/startays.md]: same

**Pull request merged**